### ISSUE TITLE
A field removed from the rules is still tested

### ISSUE THREAD

**Issue by ClemensLinnhoff** [OpenSimulationInterface/osi-validation#52]
#### Describe the bug
I also to remove a field from the rules, e.g. mounting_position_rmse from the sensor view, but I still get a warning:

```
SensorView.mounting_position_rmse.is_set(None) does not comply in SensorView
```

#### Describe how to reproduce the bug
1. Run the example trace file and the warning will appear.
2. Open rules/osi_sensorview.yml
3. Delete the line "mounting_position_rmse:"
4. Run the example trace file again. The warning will still appear.

#### Describe the expected behavior
I would expect the warning to not appear anymore, because there is no rule defining the mounting_position_rmse anymore. I am not sure, where this warning is coming from. Are the rules cached somewhere?

#### Describe the OS you are using
 - OS: Ubuntu 22.04
